storage/pgsql/schema.00001.sql

```diff
@@ -0,0 +1,7 @@
+CREATE INDEX idx_enrollment_queue_active_retrieval
+ON enrollment_queue (id, priority DESC, created_at)
+WHERE active = TRUE;
+
+CREATE INDEX idx_command_results_lookup
+ON command_results (id, command_uuid)
+INCLUDE (status, updated_at);
```

Comment:
we already have a primary key on `(id, command_uuid)`, but the `INCLUDE` here allows us to store this info in the index.

having that said, I have no problems removing if you think it's redundant